update_checker.go

```diff
@@ -21,7 +22,7 @@ type UpdateChecker struct {
 
 // NewUpdateChecker creates an update checker
 func NewUpdateChecker(updater *Updater, ctx Context, log logging.Logger) UpdateChecker {
-	return newUpdateChecker(updater, ctx, log, DefaultTickDuration())
+	return newUpdateChecker(updater, ctx, log, time.Hour)
```

Comment:
yup.. in this PR, the ticker stuff got better (default is a const): https://github.com/keybase/go-updater/pull/65